pkg/cmd/pr/shared/editable.go

```diff
@@ -7,81 +7,79 @@ import (
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/cli/cli/api"
 	"github.com/cli/cli/internal/ghrepo"
+	"github.com/cli/cli/pkg/set"
 	"github.com/cli/cli/pkg/surveyext"
 	"github.com/shurcooL/githubv4"
 )
 
 type Editable struct {
```

Comment:
The addition of `add` and `remove` was starting to make this struct excessively big which is why I decided to break it down further which resulted in much of the code changes in this PR.